### DISCUSSION TITLE
Uso de <math.h>

### DISCUSSION THREAD

**Discussion by Utmite** [IIC2133-PUC/2024-2#108] [Tarea 1] [answered]
Hola, buenas noches

Para mi tarea, necesito determinar en qué nivel se encuentra un nodo en un max-heap. Para ello, quisiera saber si se puede usar la función log2 de la biblioteca <math.h> para calcular el logaritmo en base 2.

Muchas gracias por su atención.

**Comment by tomastrivino** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/7591a565-cb52-4d28-856e-66e43abc89cd)
